conf/messages.en

```diff
@@ -1578,7 +1578,8 @@ organisationNomineesBankDetails.rollNumber = Building society roll number (if yo
 organisationNomineesBankDetails.rollNumber.hint = They can find it on their bank card, statement or passbook
 organisationNomineesBankDetails.rollNumber.error.length = Building society roll number must be between 1 and 18 characters
 organisationNomineesBankDetails.rollNumber.error.format = Roll number must only include letters, numbers, spaces, hyphens and forward slashes
-organisationNomineesBankDetails.basc.warning = We will send repayments using BACS. This is an electronic system that makes payments directly from HMRC to your charity’s account. You should make sure the charity can receive BACS payments.
+organisationNomineesBankDetails.basc.warning = We will send repayments using BACS. This is an electronic system that makes payments directly from HMRC to the charity’s account. You should make sure the charity can receive BACS payments.
+
```

Comment:
added newline here - remember to keep the line numbers the same across messages.en and messages.cy to make our life much easier when we get the Welsh translations!